common/source/docs/common-escs-and-motors.rst

```diff
@@ -49,6 +49,7 @@ Protocols
     DroneCAN ESCs <common-uavcan-escs>
     Currawong Velocity ESCs <common-velocity-can-escs>
     Hobbywing DroneCAN ESCs <common-hobbywing-dronecan-esc>
+    Hargrave Technologies DroneCAN ESCs <common-hargrave-dronecan-escs>
```

Comment:
remove this is not using a unique comm prtocol...its dronecan and listing it under dronecan escs is the correct location you will need a hidden toclist for it at the bottom of the page